tests/next_tests/integration_tests/feature_tests/ffront_tests/ffront_test_utils.py

```diff
@@ -46,10 +46,8 @@ def __gt_allocator__(
 @pytest.fixture(
     params=[
         next_tests.definitions.ProgramBackendId.ROUNDTRIP,
-        # next_tests.definitions.ProgramBackendId.GTIR_EMBEDDED, # FIXME[#1582](havogt): enable once all ingredients for GTIR are available
```

Comment:
I would like to have one backend that does GTIR fieldview and one that does ITIR-like fieldview